logdetective/server/server.py

```diff
@@ -504,16 +504,9 @@ async def collect_emoji_task():
 
 
 async def schedule_collect_emojis_task():
-    """Schedule the collect_emojis_task to run every day at midnight"""
+    """Schedule the collect_emojis_task to run every hour"""
```

Comment:
The docstring no longer matches the implementation.

```suggestion
    """Schedule the collect_emojis_task to run on a configured interval"""
```